cluster/src/main/java/org/apache/iotdb/cluster/metadata/CMManager.java

```diff
@@ -1717,7 +1726,7 @@ public MNode getMNode(MNode deviceMNode, String measurementName) {
               () -> {
                 try {
                   showTimeseries(group, plan, resultSet, context);
-                } catch (CheckConsistencyException e) {
+                } catch (CheckConsistencyException | MetadataException e) {
```

Comment:
LGTM. But there is one minor issues.  This will ignore some errors, resulting in query results have problems. Like we execute  show timeseries, some data group failed with other errors exclude tag not found, we will return result.

Reply:
1. We send requests to all DataGroups only for sequences that are not found in partitiontable.
2. The general metadata exception is because the DataGroup does not contain the sequence. Therefore, you can skip the metadata exception by default. add only the queried sequences.

Reply:
I feel like the previous design here was to ignore all MetaExceptions and read as many as we could.
If we had no intention of changing the design, it would seem pointless to catch this exception?

Reply:
Thanks , I alter this exception , Same as before.